phpcs.xml.dist

```diff
@@ -44,8 +40,17 @@
         <exclude-pattern>tests/Doctrine/Tests/ORM/Functional/ValueObjectsTest.php</exclude-pattern>
     </rule>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingAnyTypeHint">
+        <exclude-pattern>tests/*</exclude-pattern>
+    </rule>
+
+    <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint">
+        <exclude-pattern>tests/*</exclude-pattern>
+    </rule>
```

Comment:
We should enforce those rules on our tests as well. Excluding (selected) fixtures, maybe.

Reply:
It will make the merges up harder, and can be done after 3.0.0 is released. I don't remember what exactly what the plan is regarding the merges up and the release of 3.0.0, but I guess the merges up will stop or at least slow down at some point. I was planning on removing these rules then (they result in a huge diff).

Reply:
Sure, that was more like a future TODO. 👍

Reply:
Oh right, OK 👍